### PR TITLE
Ricky/timezone

### PR DESCRIPTION
**Pull request by raykhenberg**

### CODE REVIEW

File: daterangepicker.js [hunk 1]
```diff
@@ -1394,6 +1417,14 @@
             this.container.remove();
             this.element.off('.js-daterangepicker');
             this.element.removeData();
+        },
+
+        setTimeForCustomRange: function() {
+            this.setStartDate(this.startDate.startOf('day'));
```
Comment (jordangarcia) [inline daterangepicker.js:1423]: Does this imply that it's being passed a moment date object.   Is that part of the library?

File: daterangepicker.js [hunk 2]
```diff
@@ -1206,6 +1226,9 @@
                     this.clickApply();
             }
 
+            // Set to beginning of day and end of day for custom daterange.
+            this.setTimeForCustomRange();
```
Comment (jordangarcia) [inline daterangepicker.js:1230]: What if there aren't any customer time ranges?

File: daterangepicker.js (hunk 1 above)
Comment (raykhenberg) [reply daterangepicker.js:1423]: Yeah, all the dates in this plugin are wrapped in moment().

**Comment by jordangarcia** [comment]
LGTM shipit

File: daterangepicker.js (hunk 2 above)
Comment (raykhenberg) [reply daterangepicker.js:1230]: It's the default option that is automatically added.

**Pull request merged**